cookbooks/travis_python/recipes/pyenv.rb

```diff
@@ -155,7 +167,7 @@
   end
 
   execute "install packages #{py}" do
-    command "#{virtualenv_root}/#{pyname}/bin/pip install --upgrade #{packages.join(' ')}"
+    command "#{virtualenv_root}/#{pyname}/bin/pip install -I -U #{packages.join(' ')}"
```

Comment:
Why do you need the ``-I`` here? Nothing should already be installed in a brand new installation.